scripts/trash-talk.coffee

```diff
@@ -8,15 +8,15 @@
 #   None
 #
 # Commands:
-#   hubot Does <person> have an opinion about <technology>[?] - <person> talks trash about <technology>.
+#   hubot Does <person> have an opinion (about|on) <technology>[?] - <person> talks trash about <technology>.
 #
 # Author:
 #   snackCake
 
 module.exports = (robot) ->
-  robot.respond /Does (.+) have an opinion about ([^?]+)\??/i, (msg) ->
+  robot.respond /Does (.+) have an opinion (about|on) ([^?]+)\??/i, (msg) ->
```

Comment:
You could use a non-capturing group here to avoid having to change the indices below. `(?:about|on)`